en/render/camera.md

```diff
@@ -1,39 +1,135 @@
 # Camera
 
-The Camera component is useful when making a scroller game or other game that needs to move the perspective. Before when in the absence of a Camera component, the scroller in game is implemented by moving the scene node or the game root node, which will result in a matrix of nodes need to be recalculated. And the performance will naturally drop due to this. The camera renders the scene through camera matrix directly in the rendering process, it will be much more efficient than moving lots of nodes.
+The camera is the window that the player observes the game world, the scene must have at least one camera, also can have multiple cameras at the same time. When creating a scene, Creator creates a camera named `Main Camera` by default as the main camera for this scene. Multi-camera support allows you to easily implement advanced customization effects, such as the two-player split screen effect, or the creation of a small scene map.
 
-The Camera component provides two properties for the user to set:
+## Camera attribute
 
-- `targets`: Specifies which nodes the camera will render, including their child nodes.
-- `zoomRatio`: Specifies the zoom ratio of the camera, the larger the value, the larger the image.
+- cullingMask
 
-The Camera component will move along with the node it is attached to, and we can imagine the node is a hand holding the camera, which will only shoot his targets, and the FOV of the camera is the device screen size.
+`cullingMask` will determine which parts of the scene the camera is used to render. The `cullingMask` in the camera component in the **Properties** lists the mask options currently available, and you can combine them to generate `cullingMask` by selecting these options.
 
-## Example
+For example, the `cullingMask` setting in the following figure indicates that the camera is used only to render the UI part of the game, that the UI part of the game does not need to be moved, and that the game node may move out of the screen, requiring another camera to follow the game node.
```

Comment:
For example, the `cullingMask` setting in the following figure indicates that the camera is used only to render the UI part of the game. Generally game UI does not need to be moved, but the player may move out of the screen, then you need another camera to follow the player.